apple/LayoutReanimation/REASwizzledUIManager.mm

```diff
@@ -90,7 +90,9 @@ - (void)reanimated_manageChildren:(NSNumber *)containerTag
       for (REAUIView *view in permanentlyRemovedChildren) {
         [originalSelf.animationsManager endAnimationsRecursive:view];
       }
-      [originalSelf.animationsManager removeAnimationsFromSubtree:(REAUIView *)container];
+      if (![originalSelf.animationsManager hasLayoutAnimation:(REAUIView *)container]) {
+        [originalSelf.animationsManager removeAnimationsFromSubtree:(REAUIView *)container];
+      }
```

Comment:
This solution can cause a memory leaks unfortunately. I think we should call removeAnimationsFromSubtree for every view from permanentlyRemovedChildren array.

Reply:
@Latropos ⬆️ bump